plugins/CoreHome/Columns/VisitorSecondsSinceOrder.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+namespace Piwik\Plugins\CoreHome\Columns;
+
+use Piwik\Date;
+use Piwik\Plugin\Dimension\VisitDimension;
+use Piwik\Plugin\Segment;
+use Piwik\Tracker\Action;
+use Piwik\Tracker\Request;
+use Piwik\Tracker\Visitor;
+
+class VisitorSecondsSinceOrder extends VisitDimension
+{
+    protected $columnName = 'visitor_seconds_since_order';
+    protected $columnType = 'INT(11) UNSIGNED NULL';
+    protected $segmentName = 'secondsSinceLastEcommerceOrder';
```

Comment:
btw will you also add all the previous segment names again for days to not break API? Ideally we'd even keep the old dimensions (the old classes) so custom reports wouldn't break either but not sure if possible.

Reply:
Seeing the old segment name is below 👍